src/global_logic.js

```diff
@@ -574,7 +574,7 @@ module.exports.calcBasedOneSummon = function (summonind, prof, buff, totals) {
             otherCoeff *= 1.10;
         }
         if (key == "Djeeta") {
-            otherCoeff *= 1.0 + totals[key]["superATK"];
+            otherCoeff *= 1.0 + (0.01 * totals[key]["slaysnakes_myth"][0] * totals[key]["slaysnakes_myth"][1]); //[0]: amount. [1]: stacks number
```

Comment:
> //[0]: amount. [1]: stacks number

variable names can explain that comment

```js
let [amount, stack] = totals[key]["slaysnakes_myth"];
otherCoeff *= 1.0 + (0.01 * amount * stack);
```

- [ ] need calc amount * stack in calcBasedOnSummon? or addSkilldataToTotals?
  - because of the variable scope, it's on key == Djeeta context here, not only slaysnakes_myth 
    "amount" and "stack" variables can be stype == slaysnakes_myth context on addSkilldataToTotals

Reply:
Resolve and Marked as TODO (won't do in this PR)
this requires refactoring the logic with the copied source together.